Software/WinIOT/Readme.MD

```diff
@@ -0,0 +1,23 @@
+# BrickPi3
```

Comment:
For this readme header I would take off line 1 ("# BrickPi3"). In this context (already in the BrickPi3 repo), this readme is already a subset of BrickPi3, so set the header to be WinIOT (more in my comment on line 2).

Reply:
Appreciate that but this is partially for SEO and partially to help folks that link to it from the outside understand what they're looking at.